modules/ROOT/pages/clustering/setup/discovery.adoc

```diff
@@ -228,95 +232,182 @@ For more information, see <<clustering-discovery-k8s>>.
+. Then repeat steps 6 and 7 for servers 2 and 3.
+Ensure that they are set to `dbms.cluster.discovery.version=V2_OVER_V1`.
+Restart them sequentially, not in parallel.
 +
 image:discovery-service/in-place-23-v2-over-v1.png[]
 
-. Similar to step 4, verify that `stateComparison` shows `Matching`.
+. Similar to step 5, verify that `stateComparison` shows `Matching`.
 
-. Repeat steps 5, 6 and 7, restarting servers 1, 2, and 3 sequentially, with the new setting `dbms.cluster.discovery.version=V2_ONLY`
+. Repeat steps 6, 7, 8, 9, restarting servers 1, 2, and 3 sequentially, with the new setting `dbms.cluster.discovery.version=V2_ONLY`
 image:discovery-service/in-place-123-v2-only.png[]
 . Verify that `CALL dbms.cluster.showParallelDiscoveryState()` now shows `V2_ONLY` running.
 Note that `stateComparison` is `N/A` because you do not have v1 to compare states anymore.
 
 [[discovery-v1-to-v2-new-server]]
 === New server rolling
+The new server rolling requires three running servers and three new servers.
+. For all the servers, ensure that new settings are added to the configuration as detailed the xref:clustering/setup/discovery.adoc#_preparation[Preparation] section.
++
+As an example, for those using the list resolver, the settings for all the servers should include:
++
+```
+dbms.cluster.discovery.resolver_type=LIST
 
-. Ensure that the new settings are added to the configuration as detailed in the preparation section.
-. Start up three new servers with the setting `dbms.cluster.discovery.version=V1_OVER_V2`
+dbms.cluster.discovery.endpoints=server01.example.com:5000,server02.example.com:5000,server03.example.com:5000,server04.example.com:5000,server05.example.com:5000,server06.example.com:5000
+dbms.cluster.discovery.v2.endpoints=server01.example.com:6000,server02.example.com:6000,server03.example.com:6000,server04.example.com:6000,server05.example.com:6000,server06.example.com:6000
+```
+. Start up the three new servers with the setting `dbms.cluster.discovery.version=V1_OVER_V2`.
 +
 image:discovery-service/v1_over_v2.png[]
-
-. Using `bolt://`, connect to servers 4, 5, 6, and run the following procedure.
-It is important to connect via `bolt://` because otherwise the procedure might be routed and executed not on the intended server.
++
+. Using `bolt://`, connect to the system database of servers 4, 5, 6, and run the following procedure.
+This can be done using via `./cypher-shell -a bolt://localhost:7685 -d system` for example.
 +
 [source,cypher]
 ----
 CALL dbms.cluster.showParallelDiscoveryState();
 ----
 +
-The output should display `Matching` in the `stateComparison`.
-If they are not, wait and try again.
-Once matching, servers 1, 2, and 3 can be shut down.
-
-. Start up another three new servers with the setting `dbms.cluster.discovery.version=V2_OVER_V1`.
+The expected result should display `v2ServerCount` as 3.
+`stateComparison` that does show matching is fine at this point.
```

Comment:
Sentence doesn't quite make sense. A suggestion:
```suggestion
`stateComparison` is not expected to match at this stage because the original servers are not visible to the V2 discovery service.
```